packages/components/src/ListView/Items/Item/Item.component.js

```diff
@@ -66,6 +68,13 @@ class Item extends Component {
 					label={searchCriteria ? getSearchedLabel(item.label) : item.label}
 					aria-label={ariaLabel}
 				/>
+				{item.icon && (
+					<TooltipTrigger label={item.icon.title} tooltipPlacement="bottom">
+						<span>
```

Comment:
why this extra span?

Reply:
@frassinier I took it from Enumeration, without span  the icon becomes a button
![image](https://user-images.githubusercontent.com/54246061/98117631-baaf6180-1eb2-11eb-80ac-dbb6573ae3c7.png)

Reply:
it seems tooltip trigger is adding the onClick